src/main/java/org/prebid/server/auction/BidResponseCreator.java

```diff
@@ -1374,12 +1346,16 @@ private CacheAsset toCacheAsset(String cacheId) {
      * Creates {@link ExtBidPrebidVideo} from bid extension.
      */
     private ExtBidPrebidVideo getExtBidPrebidVideo(ObjectNode bidExt) {
-        final ExtPrebid<ExtBidPrebid, ObjectNode> extPrebid = getExtPrebid(bidExt);
-        final ExtBidPrebid extBidPrebid = extPrebid != null ? extPrebid.getPrebid() : null;
+        final ExtBidPrebid extBidPrebid = getExtPrebid(bidExt);
+
         return extBidPrebid != null ? extBidPrebid.getVideo() : null;
     }
 
-    private ExtPrebid<ExtBidPrebid, ObjectNode> getExtPrebid(ObjectNode bidExt) {
-        return bidExt != null ? mapper.mapper().convertValue(bidExt, EXT_PREBID_TYPE_REFERENCE) : null;
+    private ExtBidPrebid getExtPrebid(ObjectNode bidExt) {
+        if (bidExt == null) {
```

Comment:
Would it be helpful if we'll add check like `bidExt.hasNonNull(PREBID_EXT)`.
This will prevent throwing unnecessary exception.

PS. Pls verify similar code below.